test/test_braces_spacing.coffee

```diff
@@ -27,15 +27,23 @@ sources =
                    {  foo,
                       bar  } = x
                    '''
+    empty:
+        noSpaces: '{}'
+        oneSpace: '{ }'
+        twoSpaces: '{  }'
```

Comment:
I had the keys in alphabetical order. Can you update the placement to keep it in alpha order? Same for the configs below